runs/comparisons/suprb_tuning.py

```diff
@@ -81,49 +83,91 @@ def suprb_ES_GA_space(trial: Trial, params: Bunch):
+            params.solution_composition__crossover__n = trial.suggest_int(
+                'solution_composition__crossover__n', 1, 10)
 
         params.solution_composition__mutation__mutation_rate = trial.suggest_float(
             'solution_composition__mutation_rate', 0, 0.1)
 
-    experiment_name = f'ES Tuning & Experimentation {job_id} {problem}'
+        params.solution_composition__init__mixing__experience_weight = trial.suggest_float(
+            'solution_composition__init__mixing__experience_weight', 0, 2)
+        # params.solution_composition__init__mixing__experience_calculation__lower_bound = trial.suggest_float(
+        #     'solution_composition__init__mixing__experience_calculation__lower_bound', 0, 10)
+        params.solution_composition__init__mixing__experience_calculation__upper_bound = trial.suggest_float(
+            'solution_composition__init__mixing__experience_calculation__upper_bound', 20, 50)
+
+        params.solution_composition__init__mixing__filter_subpopulation = trial.suggest_categorical(
+            'solution_composition__init__mixing__filter_subpopulation',
+            ['FilterSubpopulation', 'NBestFitness', 'NRandom', 'RouletteWheel'])
+        params.solution_composition__init__mixing__filter_subpopulation = getattr(
+            mixing_model, params.solution_composition__init__mixing__filter_subpopulation)()
+
+        params.solution_composition__init__mixing__experience_calculation = trial.suggest_categorical(
+            'solution_composition__init__mixing__experience_calculation',
+            ['ExperienceCalculation', 'CapExperience', 'CapExperienceWithDimensionality'])
+        params.solution_composition__init__mixing__experience_calculation = getattr(
+            mixing_model, params.solution_composition__init__mixing__experience_calculation)()
+
+        params.solution_composition__init__mixing__experience_weight = trial.suggest_float(
+            'solution_composition__init__mixing__experience_weight', 0, 2)
+        # params.solution_composition__init__mixing__experience_calculation__lower_bound = trial.suggest_float(
+        #     'solution_composition__init__mixing__experience_calculation__lower_bound', 0, 10)
+
+        if isinstance(params.solution_composition__init__mixing__experience_calculation, mixing_model.CapExperienceWithDimensionality):
+            params.solution_composition__init__mixing__experience_calculation__upper_bound = trial.suggest_float(
+                'solution_composition__init__mixing__experience_calculation__upper_bound', 1, 5)
+        else:
+            params.solution_composition__init__mixing__experience_calculation__upper_bound = trial.suggest_float(
+                'solution_composition__init__mixing__experience_calculation__upper_bound', 20, 50)
+
+        params.solution_composition__init__mixing__filter_subpopulation__rule_amount = rule_amount
+
+    # Change this value according to run
+    rule_amount = 1
```

Comment:
I think that this should not be part of the tuning script but rather outside living in a few nested loops with the other options we all want to gridsearch

Reply:
Done

Reply:
Done